scene/gui/button.cpp

```diff
@@ -337,8 +337,18 @@ void Button::_notification(int p_what) {
 						} break;
 					}
 
-					Rect2 icon_region = Rect2(icon_ofs, icon_size);
-					draw_texture_rect(_icon, icon_region, false, icon_modulate_color);
+
```

Comment:
Static checks are failing because of one excessive newline. Click on the first [details](https://github.com/godotengine/godot/actions/runs/8992956093/job/24705168565?pr=91683) to see it yourself.
![image](https://github.com/godotengine/godot/assets/66727710/9eb15d72-746d-4c33-97d6-d1a25bdc4046)

```suggestion
```